lib/code-manager.js

```diff
@@ -126,14 +144,17 @@ export function getRegexString() {
   return regexString;
 }
 
-export function getBreakpoints() {
-  const buffer = store.editor.getBuffer();
-  const breakpoints = [buffer.getFirstPosition()];
+export function getBreakpoints(editor: atom$TextEditor) {
+  const buffer = editor.getBuffer();
+  const breakpoints = [new Point(0, 0)];
```

Comment:
I'm totally OK with the "hardcoded" version. The comment is just for future readers and our future selfs 😄